config/base/kustomization.yaml

```diff
@@ -18,6 +18,7 @@ resources:
 - 300-operator_v1alpha1_pipeline_crd.yaml
 - 300-operator_v1alpha1_trigger_crd.yaml
 - 300-operator_v1alpha1_config_crd.yaml
+- 300-operator_v1alpha1_pac_crd.yaml
```

Comment:
@sm43 as this PR focuses on adding PAC for openshift shall we move this to config/openshift folder only

Reply:
yep make sense. updated